api/spec/smithy/model/types/LoginNodesPool.smithy

```diff
@@ -0,0 +1,17 @@
+namespace parallelcluster
+
+@enum([
+    {name: "PENDING", value: "pending"},
+    {name: "ACTIVE", value: "active"},
+    {name: "FAILED", value: "failed"},
+])
+string LoginNodesState
+
+structure LoginNodesPool {
+    @required
+    status: LoginNodesState,
```

Comment:
[Nitpick] We should have the same naming. If we call `status` the field we should have a `LoginNodeStatus` to represent it.

Reply:
We have a conflict with another class name. For this reason I used `LoginNodesState`